package.json

```diff
@@ -56,6 +56,7 @@
   },
   "version": "1.19.0",
   "dependencies": {
+    "@backstage/backend-test-utils": "workspace:^",
```

Comment:
I'm not sure why this would be needed? Pretty sure you can removed this

Reply:
removed this and the changeset :)

Reply:
It still looks to be there from my end 🤔

Reply:
Also, make sure that the `yarn.lock` change is not there as well, pretty sure these two things are what are causing the builds to fail

Reply:
In the end you should only have two files with changes: `confluenceToMarkdown.examples.test.ts` and ``confluenceToMarkdown.test.ts`

Reply:
Removed from both package.json and yarn.lock :) The two confluenceToMarkdown tests are the only files changed now, thanks so much Andrew!!